chart/compass/charts/postgresql/templates/secrets.yaml

```diff
@@ -28,4 +28,33 @@ data:
   postgresql-username: {{ include "postgresql.username" . | b64enc | quote }} #TODO remove after migration to separate user will be done
   postgresql-password: {{ include "postgresql.password" . | b64enc | quote }} #TODO remove after migration to separate user will be done
   postgresql-directorDatabaseName: {{ .Values.global.database.embedded.directorDBName | b64enc | quote }} #TODO remove after migration to separate user will be done
-{{- end -}}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "postgresql.fullname" . }}
+  namespace: kyma-system
```

Comment:
If this secret is applied only for the e2e tests, probably it would make sense to use the variable for the namespace here as well. What do you think?